tests/ReplayData/LazyRepository.testGetLatestRelease.txt

```diff
@@ -0,0 +1,11 @@
+https
+GET
+api.github.com
+None
+/repos/gh_user/PyGithub/releases/latest
```

Comment:
I think all `gh_user/PyGithub` occurrences in these txt files should re replaced with `PyGithub/PyGithub` for the tests to work.